src/structures/BaseCommandInteraction.js

```diff
@@ -123,6 +125,14 @@ class BaseCommandInteraction extends Interaction {
       }
     }
 
+    if (attachments) {
+      result.attachments = new Collection();
+      for (const attachment of Object.values(attachments)) {
+        const patched = new MessageAttachment(attachment.url, attachment.filename, attachment);
+        result.attachments.set(attachment.id, patched ?? attachment);
```

Comment:
When will patched be nullable? You always create an attachment object

Reply:
It won't be I just forgot to remove this null check, fixed now